components/contact.module.css

```diff
@@ -0,0 +1,108 @@
+.contact_section_container {
+	width: 100%;
+	height: 100%;
+	display: grid;
```

Comment:
I was impressed by using grid for this section!  I will try to use grid next time!!

Reply:
Agreed! Great responsive design.